mne_bids/tests/test_mne_bids.py

```diff
@@ -54,14 +55,16 @@ def test_fif():
                 task=task, raw_file=raw_fname, events_data=events_fname,
                 output_path=output_path, event_id=event_id, overwrite=True)
 
-    # let's do some modifications to meas_date
+    # give the raw object some fake participant data
     raw = mne.io.read_raw_fif(raw_fname)
-    raw.anonymize()
+    #raw.anonymize()
```

Comment:
what was this testing and why did you remove it?

Reply:
I have no idea. I guess if @teonbrooks wrote it then they had a reason?
Instead of removing I could get it to do the bids conversion a 3rd time maybe and apply the subject data as I did...